src/tests/test_cases/sections.py

```diff
@@ -75,7 +90,7 @@ def no_blank_line_after_section():
 
     Returns
     -------
-    A whole lot of values.
+    A value of some sort.
```

Comment:
I think the "blank line after a section" should not be enforced for the last section of the docstring (as discussed in #230).